src/components/ec-timer/ec-timer.vue

```diff
@@ -78,18 +93,42 @@ const diameter = computed(() => radius.value * 2 + strokeWidth.value);
 const viewbox = computed(() => `0 0 ${diameter.value} ${diameter.value}`);
 const circumference = computed(() => 2 * Math.PI * radius.value);
 const steps = computed(() => circumference.value / props.seconds);
-const offset = computed(() => circumference.value + steps.value * secondsLeft.value);
+const offset = computed(() => circumference.value + steps.value * totalSecondsLeft.value);
 
 const radius = ref(24);
 const strokeWidth = ref(4);
-const secondsLeft = ref(props.seconds);
+const minutesLeft = ref(getMinutes(props.seconds));
+const secondsLeft = ref(getSeconds(props.seconds));
```

Comment:
these variables are computed. We operate only with totalSecondsLeft.

```suggestion
const minutesLeft = computed(()=>  props.showMinutes 
  ?  Math.floor(totalSecondsLeft.value / 60);
  : null);
const secondsLeft = computed(()=>  props.showMinutes
   ? totalSecondsLeft.value - Math.round(minutesLeft.value * 60);
  : totalSecondsLeft);
```